apmpackage/apm/0.1.0/data_stream/app_metrics/manifest.yml

```
@@ -1,5 +1,5 @@
 title: APM application metrics
 type: metrics
-dataset: apm
+dataset: apm.user
```

```suggestion
dataset: apm.app
```

The title of the stream is "app metrics" after all :)

---

ha, I was 50/50 between "app" and "user"